tests/lang/core.pk

```diff
@@ -50,6 +51,22 @@ for i in 0..1000
   assert(abs(sin(i) / cos(i) - tan(i)) < threshold)
 end
 
+assert(abs(cosh(PI/2) - 2.5) < 0.01 )
+assert(abs(tanh(PI/4) - 1.0) < 0.5 )
```

Comment:
Don't use a very large margin (.5, .01), instead use the threshold here. get the value using another language (python if you have).
Ex: `assert((cosh(.5) - 1.1276259652063807) < threshold)` where you can get he value `1.1276...` from python or seme other language.

Reply:
Done! Thank you!